### DISCUSSION TITLE
--ignore option for wsl --export ?

### DISCUSSION THREAD

**Discussion by jonrbates** [microsoft/WSL#8000] [Q&A]
Hi all, is there a recommendation for ignoring a path in the export?
```wsl --export ub ./ub.tar --ignore /home/nobody/.cache/lots_o_files```
thx!

**Comment by Biswa96** [comment]
Instead exclude the file while creating the tar file with `--exclude` option with tar command.

**Reply by jonrbates** [reply]
Thank you, but wouldn't I need to add the --exclude to the source of wsl --export ? 
wsl --export already outputs a .tar, so an external pipe won't work, unless I'm missing something.
I'm ok to modify the source if you know where it is 🙏
